packages/core/admin/ee/admin/pages/SettingsPage/pages/SingleSignOn/index.js

```diff
@@ -189,6 +192,48 @@ export const SingleSignOn = () => {
                       ))}
                     </Select>
                   </GridItem>
+                  <GridItem col={6} m={6} s={12}>
+                    <MultiSelect
+                      disabled={!canUpdate}
+                      hint={formatMessage({
+                        id: 'Settings.sso.form.localAuthenticationLock.description',
+                        defaultMessage:
+                          'Select the roles for which you want to disable the local authentication',
+                      })}
+                      error={
+                        formErrors.ssoLockedRoles
+                          ? formatMessage({
+                              id: formErrors.ssoLockedRoles.id,
+                              defaultMessage: formErrors.ssoLockedRoles.id,
+                            })
+                          : ''
+                      }
+                      label={formatMessage({
+                        id: 'Settings.sso.form.localAuthenticationLock.label',
+                        defaultMessage: 'Local authentication lock-out',
+                      })}
+                      name="ssoLockedRoles"
+                      onChange={(value) => {
+                        handleChange({ target: { name: 'ssoLockedRoles', value } });
+                      }}
+                      placeholder={formatMessage({
+                        id: 'components.InputSelect.option.placeholder',
+                        defaultMessage: 'Choose here',
+                      })}
+                      onClear={() => {
+                        const emptyArray = [];
+                        handleChange({ target: { name: 'ssoLockedRoles', emptyArray } });
+                      }}
+                      value={modifiedData.ssoLockedRoles || []}
+                      withTags
+                    >
+                      {roles.map(({ id, name }) => (
+                        <MultiSelectOption key={id} value={id.toString()}>
```

Comment:
Why do you have to convert it to a string?

Reply:
because in the form we handled the id in the MultiSelect and inside the Defaul role dropdown as strings